components/TxList/index.tsx

```diff
@@ -212,10 +216,15 @@ const TxList: React.FC<TxListProps & { maxCount?: string; pageSize?: number }> =
                     )}
                   </td>
                   <td>
-                    <Address address={from} type={item.from_account?.type} />
+                    <Address
+                      address={from}
+                      type={item.from_account?.type}
+                      domain={from_alias}
```

Comment:
Seems that only the transaction list adopts `domain`, but it's expected to be displayed all over the page, except the title of an account page.

On account page, the domain will be displayed next to the address instead of replacing it.